index.d.ts

```diff
@@ -21,7 +21,7 @@ export type ColorSupportLevel = 0 | 1 | 2 | 3;
 /**
 Detect whether the terminal supports color.
 */
-export interface ColorSupport {
+export type ColorSupport = {
```

Comment:
> This is not the same. What was the XO error message about?

`3:18  Use a type instead of an interface.                                      @typescript-eslint/consistent-type-definitions`